conjure-core/src/main/java/com/palantir/conjure/parser/types/names/TypeName.java

```diff
@@ -34,8 +37,11 @@
 public abstract class TypeName {
 
     private static final Pattern CUSTOM_TYPE_PATTERN = Pattern.compile("^[A-Z][a-z0-9]+([A-Z][a-z0-9]+)*$");
-    static final ImmutableSet<String> PRIMITIVE_TYPES =
-            ImmutableSet.of("any", "string", "integer", "double", "boolean", "safelong", "rid", "bearertoken", "uuid");
+    static final Set<String> PRIMITIVE_TYPES = Arrays.stream(PrimitiveType.Value.values())
```

Comment:
The values static method is simpler and avoids the UNKNOWN check.
```suggestion
    static final Set<String> PRIMITIVE_TYPES = PrimitiveType.values().stream()<..etc>
```

Reply:
👍